src/StatusQ/Core/Utils/Emoji.qml

```diff
@@ -14,17 +14,26 @@ QtObject {
         "small": "18x18",
         "verySmall": "16x16"
     }
+    readonly property var format: {
+        "png": "png",
```

Comment:
Why do we need .png format handling?

Reply:
Interesting.. https://tinyurl.com/y33jc4c9
![image](https://user-images.githubusercontent.com/33099791/168325298-1bec4738-50f1-4c6c-93f4-3b92eedd284a.png)